apps/bot_manager/lib/spline_path.ex

```diff
@@ -0,0 +1,87 @@
+defmodule SplinePath do
+  @moduledoc """
+  This module defines methods to generate a spline path out of a waypoint path
+
+  Based on https://qroph.github.io/2018/07/30/smooth-paths-using-catmull-rom-splines.html
+  """
+  alias BotManager.Math.Vector
+
+  @segment_point_amount 5
+  @tension 0.2
+  @alpha 0.5
```

Comment:
you can also play with these values. I used the values suggested in the article, and played around with the `segment_point_amount`. After 5 I didn't see a difference